js/src/components/SnippetPreviewSection.js

```diff
@@ -17,6 +17,22 @@ const Section = styled( StyledSection )`
 	}
 `;
 
+/**
+ * Function to map the editor data to data for the preview.
+ *
+ * @param {Object} mappedData The default mappedData object.
```

Comment:
What do you mean by 'default'? 
I also think the data object you're passing to this function doesn't need to be mapped beforehand. This function can handle/should be able to handle any data object containing a title, url and description.

Proposal: `@param {Object} data The data object.`